.github/vale/dicts/aiven.dic

```diff
@@ -77,8 +77,6 @@ Instana
 io
 IPsec
 Java
-JMX
```

Comment:
Hi @bcrisologo, why are these rows and the service integration files been removed?

Reply:
Thanks Francesco! I'm not sure why they're missing since I just forked the repo, but I've updated the missing entry of JMX and hope this fixes the errors I've been getting